upstream/composer.json

```
@@ -13,7 +13,7 @@
         }
     ],
     "require": {
-        "php": ">=8.1",
+        "php": ">=8.2",
```

```suggestion
        "php": ">=8.3",
```
See https://github.com/az-digital/az-quickstart-scaffolding/pull/167